definitions/artifacts/azure-databricks-workspace.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://schemas.massdriver.cloud/definitions/artifacts/azure-databricks-workspace.json",
+  "$md": {
+    "access": "public"
+  },
+  "type": "object",
+  "title": "Azure Databricks Workspace",
+  "description": "",
+  "additionalProperties": false,
+  "required": [
+    "data",
+    "specs"
+  ],
+  "properties": {
+    "data": {
+      "title": "Artifact Data",
+      "type": "object",
+      "required": [
+        "infrastructure",
+        "authentication",
```

Comment:
`authentication` and `security` here under `required` will cause a validation failure since they aren't defined below. You likely need one of the two.

Authentication will be for if you need credentials (username and password) to access the workspace.
Security will be for if you need IAM to access the workspace.